include/deal.II/lac/relaxation_block.h

```diff
@@ -151,6 +151,16 @@ class RelaxationBlock :
      * Return the memory allocated in this object.
      */
     std::size_t memory_consumption() const;
+
+    /**
+     * Temporary ghost vector that is used in the Relaxation method when
+     * performing parallel MPI computations.
+     */
+#ifdef DEAL_II_WITH_TRILINOS
+    TrilinosWrappers::MPI::Vector *temp_trilinos_ghost_vector;
```

Comment:
Can you document what this points to? I don't see it from the rest of this patch. I assume it points to some other vector, because I don't see a memory allocation operation.